patches/src/tinycthread.c.patch

```diff
@@ -41,7 +41,7 @@ index b0ec8e95..f9717c3a 100644
  {
  #if !defined(_TTHREAD_WIN32_)
 -  return nanosleep(duration, remaining);
-+#ifdef _ZOS
++#ifdef __MVS__
```

Comment:
There is a nanosleep in zoslib, can you use that?

Reply:
I have pushed the changes, can you please review